theme/_login-theme-custom.scss

```diff
@@ -0,0 +1,65 @@
+.hospital-dest-title {
+  font-weight: bold;
+  line-height: 0.5em;
+  font-size: 2.5em;
+}
+
+#login_submit {
+  width: 100%;
+}
+
+.button-container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+.login-submit-btn-div {
+  position: relative;
+}
+
+.password_error {
```

Comment:
I think the naming of these password_* classes could have utilized the BEM convention (used by USWDS and the rest of the codebase) to a greater degree...

I'm also seeing inconsistency in the usage of dash and underscore in naming... I'll spend some time looking at both the JS validation file and how these are used within that to see if maybe things could be cleaned up/simplified here...